pkg/async/redis/cleaner.go

```diff
@@ -61,15 +61,15 @@ func (e *engine) defaultClean(
 				if err := e.cleanActiveTaskQueue(
 					ctx,
 					workerID,
-					getActiveTaskQueueName(workerID),
+					e.getActiveTaskQueueName(workerID),
```

Comment:
All changes to the async package should probably be PR'ed to deis/async or else #560 will undo much of this PR.

Reply:
Yeah, I think this was in flight before that move happened. I'll go ahead and send a PR there. Would it be better to revert the async here?

Reply:
I'm thinking make the PR to deis/async first. We'll get that merged and do a minor release. I'll update #560 to use async 1.1, get that merged, and then you can rebase this PR and all the async changes will just go away when the merge conflict is resolved.